### PR TITLE
Add the ability to connect to existed Chrome

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: lib/debug/server_cdp.rb [hunk 1]
```diff
@@ -44,17 +57,37 @@ def setup_chrome addr
...
+                            url: "devtools://devtools/bundled/inspector.html?ws=#{addr}",
+                            frameId: f_id
                           }
           when res['method'] == 'Page.loadEventFired'
             break
           end
         end
-        pid
+        DEBUGGER__.warn <<~EOS
+        Run the following command if you want to debug again:
```
Comment (ko1) [inline lib/debug/server_cdp.rb:75]: "with opened Chrome process" ?

File: README.md [hunk 2]
```diff
@@ -433,6 +433,19 @@ Type `devtools://devtools/bundled/inspector.html?ws=127.0.0.1:43633` in the addr
 
 Also `open chrome` command works like `open vscode`.
 
+Debugger tries to open Chrome automatically in some platforms and can attach to opened one from the second time unless you close it.
```
Comment (ko1) [inline README.md:436]: "close it." -> "close the chrome browser".

File: README.md [hunk 3]
```diff
@@ -433,6 +433,22 @@ Type `devtools://devtools/bundled/inspector.html?ws=127.0.0.1:43633` in the addr
...
+```
+$ rdbg target.rb --open=chrome
+DEBUGGER: Debugger can attach via TCP/IP (127.0.0.1:54004)
+DEBUGGER: wait for debugger connection...
+Launching: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=0 --no-first-run --no-default-browser-check --user-data-dir=/var/folders/kv/w1k6nh1x5fl7vx47b2pd005w0000gn/T/d20220102-83390-huc0yn
+DEBUGGER: Run the following command if you want to debug again with opened Chrome browser's process:
+
+    RUBY_DEBUG_REMOTE_DEBUGGING_PORT=54005/devtools/browser/658084a0-3d64-4049-ae2b-5e663c19685e rdbg --open=chrome target.rb
+
+*NOTE*: DO NOT use `RUBY_DEBUG_REMOTE_DEBUGGING_PORT` without opened one.
```
Comment (ko1) [inline README.md:447]:
```suggestion
NOTE: DO NOT use `RUBY_DEBUG_CRHOME_REMOTE_DEBUGGING_PORT` after closing the opened browser.
```

File: README.md [hunk 4]
```diff
@@ -433,6 +433,22 @@ Type `devtools://devtools/bundled/inspector.html?ws=127.0.0.1:43633` in the addr
...
 Also `open chrome` command works like `open vscode`.
 
+Debugger tries to open Chrome browser automatically in some platforms and can attach to opened one from the second time unless you close the Chrome browser.
+
+```
+$ rdbg target.rb --open=chrome
+DEBUGGER: Debugger can attach via TCP/IP (127.0.0.1:54004)
+DEBUGGER: wait for debugger connection...
+Launching: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=0 --no-first-run --no-default-browser-check --user-data-dir=/var/folders/kv/w1k6nh1x5fl7vx47b2pd005w0000gn/T/d20220102-83390-huc0yn
+DEBUGGER: Run the following command if you want to debug again with opened Chrome browser's process:
```
Comment (ko1) [inline README.md:443]:
```suggestion
DEBUGGER: Use a following environment variable if you want to use the opened browser again:
```

File: README.md (hunk 3 above)
Comment (simi) [reply README.md:447]: `RUBY_DEBUG_CHROME_REMOTE_DEBUGGING_PORT`

**Pull request closed without merging**